src/fsharp/service/ServiceAnalysis.fs

```diff
@@ -70,90 +85,117 @@ module UnusedOpens =
                  if firstId.idText = MangledGlobalName then 
                      None
                  else
-                     Some { Modules = openDecl.Modules |> List.map ModuleGroup.Create
+                     Some { OpenedGroups = openDecl.Modules |> List.map OpenedModuleGroup.Create
                             Range = range
                             AppliedScope = openDecl.AppliedScope }
              | _ -> None)
 
+    /// Only consider symbol uses which are the first part of a long ident, i.e. with no qualifying identifiers
     let filterSymbolUses (getSourceLineStr: int -> string) (symbolUses: FSharpSymbolUse[]) : FSharpSymbolUse[] =
         symbolUses
         |> Array.filter (fun su ->
              match su.Symbol with
              | :? FSharpMemberOrFunctionOrValue as fv when fv.IsExtensionMember -> 
-                // extension members should be taken into account even though they have a prefix (as they do most of the time)
+                // Extension members should be taken into account even though they have a prefix (as they do most of the time)
                 true
              | _ -> 
```

Comment:
Would it make sense to filter out more symbol uses here? All local values, parameters and type parameters could be filtered out before calling `QuickParse` methods and getting document line strings. Symbol bindings could also be filtered but there're several known cases where a symbol use is considered as binding even though it's from a usage.

Reply:
Yes, absolutely, good feedback , thanks

Reply:
done, thanks